test/test_docker_api_deploy.py

```diff
@@ -187,22 +202,106 @@ def test_docker_api_deploy_fail_path_in_bucket(self):
             exception_thrown = True
         assert exception_thrown
 
-    def validate_file_on_bucket_fs(self, bucket_name: str, expected_file_path: str):
-        url = "http://w:{password}@{host}:{port}/{bucket}".format(
-            host=self.docker_environment.database_host,  # type: ignore
-            port=self.docker_environment.ports.bucketfs,  # type: ignore
-            bucket=bucket_name,
-            password=self.docker_environment.bucketfs_password,  # type: ignore
-        )
-        cmd = ["curl", "--silent", "--show-error", "--fail", url]
-        p = subprocess.run(cmd, capture_output=True)
-        p.check_returncode()
-        found_lines = [
-            line
-            for line in p.stdout.decode("utf-8").split("\n")
-            if line == expected_file_path
+    # def check_file_in_bucketfs(
+    #     self,
+    #     url: str,
+    #     bucket_name: str,
+    #     bucketfs_username: str,
+    #     bucketfs_password: str,
+    #     path: str,
+    #     expected_file: str,
+    #     compression: bool,
+    # ):
+    #     CREDENTIALS = {
+    #         bucket_name: {"username": bucketfs_username, "password": bucketfs_password}
+    #     }
+    #
+    #     bucketfs = Service(url, CREDENTIALS)
+    #     bucket = bucketfs[bucket_name]
+    #
+    #     with TemporaryDirectory() as tmpdir:
+    #         file = as_file(
+    #             bucket.download(f"{path}/{expected_file}"),
+    #             filename=Path(tmpdir) / expected_file,
+    #         )
+    #
+    #         tar_mode = "r:gz" if compression else "r:"
+    #         with tarfile.open(name=file, mode=tar_mode) as tf:
+    #             tf_members = tf.getmembers()
+    #             last_tf_member = tf_members[-1]
+    #             assert last_tf_member.name == "exasol-manifest.json"
+    #             assert last_tf_member.path == "exasol-manifest.json"
```

Comment:
This implementation of the check for some reason does not work if compression=True, as the downloaded file has name "....tar.gz", but is unzipped.
On the other hand, the implementation using `curl`, see below, does work.